lighthouse-core/core.js

```diff
@@ -35,64 +33,8 @@ const log = require('./lib/log');
 
 class Core {
   static audit(artifacts, audits) {
```

Comment:
at this point, I'd be in favor of moving this `map` into runner.js and getting rid of `core.js` altogether